.eslintrc

```diff
@@ -1,3 +1,6 @@
 {
+  "parserOptions": {
+    "ecmaVersion": 2018,
+  },
```

Comment:
Required to fix eslint errors around using object spread syntax

Reply:
I think using babel-eslint@11 would solve this - https://github.com/babel/babel-eslint#breaking-changes-in-v11xx

It uses our babelrc instead of a prebaked one. If babel-eslint@next works for this, I'd prefer that solution over this one.